docs/metrics/v3.md

```diff
@@ -20,378 +22,384 @@ Some metrics are bucket specific. These will have a `/bucket` component in their
-| `/ilm` | Metrics related to ILM functionality |
+### Cluster metrics
 
-### Logger webhook metrics
+Metrics about an entire MinIO cluster.
+
+| Path                     | Description                    |
+|--------------------------|--------------------------------|
+| `/cluster/config`        | Cluster configuration metrics. |
+| `/cluster/erasure-set`   | Erasure set metrics.           |
+| `/cluster/health`        | Cluster health metrics.        |
+| `/cluster/iam`           | Cluster iam metrics.           |
+| `/cluster/usage/buckets` | Object statistics by bucket.   |
+| `/cluster/usage/objects` | Object statistics.             |
+
+#### `/cluster/config`
+
+| Name                                   | Description                                                  | Labels |
+|----------------------------------------|--------------------------------------------------------------|--------|
+| `minio_cluster_config_rrs_parity`      | Reduced redundancy storage class parity. <br><br>Type: gauge |        |
+| `minio_cluster_config_standard_parity` | Standard storage class parity. <br><br>Type: gauge           |        |
+
+#### `/cluster/erasure-set`
+
+| Name                                             | Description                                                                                            | Labels           |
+|--------------------------------------------------|--------------------------------------------------------------------------------------------------------|------------------|
+| `minio_cluster_erasure_set_overall_write_quorum` | Overall write quorum across pools and sets. <br><br>Type: gauge                                        |                  |
+| `minio_cluster_erasure_set_overall_health`       | Overall health across pools and sets (1=healthy, 0=unhealthy). <br><br>Type: gauge                     |                  |
+| `minio_cluster_erasure_set_read_quorum`          | Read quorum for the erasure set in a pool. <br><br>Type: gauge                                         | `pool_id`, `set_id` |
+| `minio_cluster_erasure_set_write_quorum`         | Write quorum for the erasure set in a pool. <br><br>Type: gauge                                        | `pool_id`, `set_id` |
+| `minio_cluster_erasure_set_online_drives_count`  | Count of online drives in the erasure set in a pool. <br><br>Type: gauge                               | `pool_id`, `set_id` |
+| `minio_cluster_erasure_set_healing_drives_count` | Count of healing drives in the erasure set in a pool. <br><br>Type: gauge                              | `pool_id`, `set_id` |
+| `minio_cluster_erasure_set_health`               | Health of the erasure set in a pool (1=healthy, 0=unhealthy). <br><br>Type: gauge                      | `pool_id`, `set_id` |
+| `minio_cluster_erasure_set_read_tolerance`       | No of drive failures that can be tolerated without disrupting read operations. <br><br>Type: gauge     | `pool_id`, `set_id` |
```

Comment:
```suggestion
| `minio_cluster_erasure_set_read_tolerance`       | Number of drive failures that can be tolerated without disrupting read operations. <br><br>Type: gauge     | `pool_id`, `set_id` |
```

Alternatively, if space is constrained, then use `No.` with a `.`. Otherwise, this might be difficult for those with less familiarity with English.

Same on the next line.

Reply:
On this, all other sections use `number`, so I'd do that here, too.